libraries/package-extractor/src/tests/PackageExtractor.test.ts

```diff
@@ -0,0 +1,155 @@
+import path from 'path';
+
+import { FileSystem, Terminal, StringBufferTerminalProvider } from '@rushstack/node-core-library';
+import { loadRushProjectFromConfigurationFile } from './Utils';
+import {
+  PackageExtractor,
+  type IExtractorProjectConfiguration,
+  type IExtractorDependencyConfiguration
+} from '../PackageExtractor';
+
+describe(PackageExtractor.name, () => {
+  // because this files will be compiled to js, and the path will change if using __dirname
+  // using path inside src folder
+  const testRepoRoot = path.join(path.resolve(__dirname, '../../'), 'src/tests/package-extractor-test-repo');
+  const testRepoBaseTargetFolder = path.join(testRepoRoot, 'output');
+  const projects = loadRushProjectFromConfigurationFile(path.resolve(testRepoRoot, 'rush.json'));
+  const terminal = new Terminal(new StringBufferTerminalProvider());
+  const packageExtractor = new PackageExtractor();
+  const projectConfigurations: IExtractorProjectConfiguration[] = projects.map((p) => ({
+    projectFolder: p.projectFolder,
+    projectName: p.packageName
+  }));
+  beforeAll(() => {
+    FileSystem.ensureEmptyFolder(testRepoBaseTargetFolder);
+  });
+  afterAll(() => {
+    // clean up target folder after each test run
+    // FileSystem.ensureEmptyFolder(testRepoBaseTargetFolder);
+  });
```

Comment:
Intentionally commented out?

Reply:
No, this line should be kept to clean test files generated in unit tests, will fix it